### ISSUE TITLE
Show lvm backups from archive even when on brtfs

### ISSUE THREAD

**Issue by tlaurion** [tasket/wyng-util-qubes#32]
Got scared lol! I pruned all but old session and thought that old lvm sessions got wiped, not nope! Just hidden. 

After sending a btrfs backup session over old lvm archive, only btrfs new sent sessions are showed by wyng-util-qubes, where wyng shows the old lvm backups.

**Comment by tasket** [comment]
@tlaurion I don't think this is lvm<->btrfs thing at all. They should be visible regardless of the volume naming format.

I think this could be due to backups made directly with `wyng send`, in which case there is no accompanying 'wyng-qubes-metadata' volume. Also, if you prune-out sessions of that volume, then the corresponding VM sessions will be gone in the util even if the VM's data volumes have those sessions.

**Comment by tasket** [comment]
BTW, autopruning won't do any thinning-out of volumes named 'wyng-*-metadata'.  You should expect sessions of that volume to only be pruned when a global (prune --all) type of command has been issued.... at least prune commands from the util will avoid doing otherwise.  So general advice is to never directly tell wyng to target the metadata volume to remove old sessions, but pruning for 'all' that includes the metadata is OK because then that session's metadata is no longer needed anyway.

The necessity of recording (& using) the Qubes xml metadata for util operation means that sessions themselves become more critical to data accessibility.  A session becomes more like a qvm-backup archive file; poking a hole in it in the wrong place could invalidate it.

Its also true that workaround options are possible, but I'm not sure what the overall value is in that case. You can still obtain the data directly with Wyng.  You could even do this: Use the util to restore the VM from a _different_ session to obtain the VM settings, then use Wyng directly to get the volume data from the desired session.  OTOH, an option could be added to the util that would restore VM settings only, making the above type of workaround faster to execute.

**Comment by tlaurion** [comment]
@tasket well, it's not the case

```
[user@dom0 ~]
$ sudo wyng --dest qubes-ssh://$wyng_qube:root@$ssh_server$archive_path --authmin 1080 --all --opt-ssh=-C --dedup list | grep volume | awk -F " " {'print $4'} | wc -l
135
[user@dom0 ~]
$ sudo wyng-util-qubes --dest qubes-ssh://$wyng_qube:root@$ssh_server$archive_path --authmin 1080 --includes -w remap -w opt-ssh=-C --dedup list --all |grep ':' | awk -F ":" {'print $1'} | wc -l
26

```

**Comment by tlaurion** [comment]
> BTW, autopruning won't do any thinning-out of volumes named 'wyng-*-metadata'. You should expect sessions of that volume to only be pruned when a global (prune --all) type of command has been issued.... at least prune commands from the util will avoid doing otherwise. So general advice is to never directly tell wyng to target the metadata volume to remove old sessions, but pruning for 'all' that includes the metadata is OK because then that session's metadata is no longer needed anyway.
> 
> The necessity of recording (& using) the Qubes xml metadata for util operation means that sessions themselves become more critical to data accessibility. A session becomes more like a qvm-backup archive file; poking a hole in it in the wrong place could invalidate it.
> 
> Its also true that workaround options are possible, but I'm not sure what the overall value is in that case. You can still obtain the data directly with Wyng. You could even do this: Use the util to restore the VM from a _different_ session to obtain the VM settings, then use Wyng directly to get the volume data from the desired session. OTOH, an option could be added to the util that would restore VM settings only, making the above type of workaround faster to execute.

I don't use autopruning, I use:
`sudo wyng --dest qubes-ssh://$wyng_qube:root@$ssh_server$archive_path --opt-ssh=-C --authmin 1080 --all prune --all-before --session $(date +%Y%m%d-%H%M%S) -u`

**Comment by tlaurion** [comment]
So I have to clone qubes of the same name and then restore each volume in a loop if I understand well.
No problem, I guess its ok since not all of those were sent as backups with wyng-util-qubes.

Non-issue I guess for people of qubesos that will only use wyng-util-qubes. I guess wyng-util-qubes only lacks arch-init then so that wyng is never used directly again for newcomers whatever origin partition scheme?

**Comment by tasket** [comment]
If you do `wyng list wyng-qubes-metadata` the sessions shown for that volume will define what the util can access.

It would be possible to add a raw volume-based listing to the util, which would be like a pass-through unfiltered.  I don't know what the user would do with that, apart from trying `wyng receive` directly.

**Comment by tasket** [comment]
This issue should be resolved now.  If not, post a new comment with status.
